enos/enos-variables.hcl

```diff
@@ -52,6 +52,18 @@ variable "alb_listener_api_port" {
   default     = 9200
 }
 
+variable "project_name" {
+  description = "The description of the project"
+  type        = string
+  default     = "boundary-enos-integration"
+}
+
+variable "tags" {
+  description = "Tags that will be applied to infrastructure resources that support tagging"
+  type        = map(string)
+  default     = null
```

Comment:
I wonder if this default value should rather be: `{}`. Not sure if merge function works well with a `null` argument for any of its args. Maybe you could just double check that it works with null.

Reply:
it does seem to work with default `null`, as we have the same setup in Vault scenarios.